rero/rero-ils
# dependencies: update to ES7 and invenio 3.3

* Updates ES version to 7.9.0.
* Updates invenio to version to 3.3.

Co-Authored-by: Peter Weber <peter.weber@rero.ch>

## Why are you opening this PR?

-We need ES 7 to implement ES cluster in kubernets

## Dependencies

## How to test?

- run scripts bootstrap and setup (maybe the virtual python env and static folder must be deleted)
- see everything works as expected

## Code review check list
